pkg/db-repo/repository.go

```diff
@@ -357,6 +371,120 @@ func ignoreCreatedAtIfNeeded(scope *gorm.Scope) {
 	}
 }
 
+// parses the []uint8 (from mysql) for field x, if there is a field xSet with type []string
+func GormParseMysqlSetType(scope *gorm.Scope) {
+	vo := reflect.ValueOf(scope.Value)
+	if vo.Kind() == reflect.Invalid {
```

Comment:
I think you can get rid of this check as you later check whether you have a struct and if not just return

Reply:
i do have that in case i have to iterate over a slice which can be nil, so it is still needed.

Reply:
If the kind is Invalid, we won't enter any of the other if-blocks or loops, will eventually call `parseMysqlSetType` and there see the kind is not a struct and return. So I don't think this block really changes anything.